README.md

```diff
@@ -247,6 +248,52 @@ Note that:
 - Either `url`, `getUrl`, or `urlPath` must be present.
 - `html` type uses `iframe` and can display `.pdf` if supported by browser.
 
+### <a name="shortcuts"></a>Shortcuts
+
+Configuration for the shortcuts in the document based on focused fields.
+
+```
+{
+ shortcut-name: {
```

Comment:
@harunurhan For the bug this is the problem as described in this post:
http://stackoverflow.com/questions/6761169
I am trying to set explicit focus on checkbox on click.